rails_event_store_active_record/spec/spec_helper.rb

```diff
@@ -1,26 +1,55 @@
+if ENV['CODECLIMATE_REPO_TOKEN']
+  require 'simplecov'
+  SimpleCov.start
+end
+
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'rails_event_store_active_record'
 
+ENV['DATABASE_URL'] ||= "postgres://localhost/rails_event_store_active_record?pool=5"
+
 RSpec.configure do |config|
+  config.failure_color = :magenta
   config.around(:each) do |example|
-    config.order = :random
-    Kernel.srand config.seed
-
-    ActiveRecord::Base.establish_connection(adapter: 'sqlite3', database: ':memory:')
+    ActiveRecord::Base.establish_connection(ENV['DATABASE_URL'])
     ActiveRecord::Schema.define do
       self.verbose = false
-      create_table(:event_store_events) do |t|
+
+      enable_extension "plpgsql"
+      enable_extension "pgcrypto"
+
+      create_table(:event_store_events_in_streams, force: true) do |t|
         t.string      :stream,      null: false
-        t.string      :event_type,  null: false
-        t.string      :event_id,    null: false
-        t.text        :metadata
-        t.text        :data,        null: false
+        t.integer     :position,    null: true
+        if ENV['DATABASE_URL'].start_with?("postgres")
+          t.references :event, null: false, type: :uuid
```

Comment:
Do we actually need the :uuid? Maybe :string would suffice? With :string we don't have to enable pgcrypto

Reply:
And what's wrong with pgcrypto?

Reply:
Nothing wrong. It's not enabled by default. Are we going to force users to enable it even though it's not really needed?

Reply:
Yep :)

Reply:
🤔